From df683eee28c07974818c0d87ad69780e5e8374a8 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Mon, 7 Mar 2022 12:06:16 +0100
Subject: [PATCH 087/104] Revert "bfq: reset entity->prio_changed in
 bfq_init_entity()"

This reverts commit 4fef3e257de23a07319512e4e7444a2adc8283bb.

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 block/bfq-cgroup.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/block/bfq-cgroup.c b/block/bfq-cgroup.c
index 402f34820..64eb39c67 100644
--- a/block/bfq-cgroup.c
+++ b/block/bfq-cgroup.c
@@ -427,7 +427,6 @@ void bfq_init_entity(struct bfq_entity *entity, struct bfq_group *bfqg)
 
 	entity->weight = entity->new_weight;
 	entity->orig_weight = entity->new_weight;
-	entity->prio_changed = 0;
 	if (bfqq) {
 		bfqq->ioprio = bfqq->new_ioprio;
 		bfqq->ioprio_class = bfqq->new_ioprio_class;
-- 
2.35.1.354.g715d08a9e5

